OpenTreeMap/otm-core
# Unify CSV columns

Connects #2531

If using `debugcelery.sh` you may need https://github.com/OpenTreeMap/otm-cloud/pull/188

Testing:

0. Run migrations

1. Go to your `latreemap` 

1. Go to the "Bulk Uploader" page. Imports with status "Verification Complete" (both tree and species) should:
  * be in the "Finished Imports" section
  * have no "Export" link
  * have no "Add to Tree Map" link when you click "View"

2. Download the template for trees and for species. The CSV column headers should:
  * be in a logical order
  * be capitalized, with spaces separating words
  * contain custom fields (but not Stewardship), prefixed with either "Planting Site:" or "Tree:"

4. Test importing a tree export:
  * Export all trees in the "West Whittier-Los Nietos" neighborhood (80 trees, 82 planting sites)
  * The CSV column headers should match the template (except now including Stewardship and Alerts)
  * Import the csv you exported. It should fail, saying that alerts and stewardship are unrecognized fields
  * Delete the Stewardship and Alerts columns from your csv, and import again. It should succeed, with 82 rows ready to add.
  * Click "Add to Tree Map". It should succeed.
  * Click the "Export" button next to your finished import. It should succeed (though the field names are lowercased)

3. Test importing a species export:
  * Export all species. The CSV column headers should match the template.
  * Import the csv you exported. It should succeed.
  * Click "View". It should show 758 species "Successfully Added".
  * Click the "Export" button next to your finished import. It should succeed (though the field names are lowercased)
